src/Omnipay/Common/Helper.php

```diff
@@ -76,16 +76,18 @@ public static function validateLuhn($number)
      * Parameters are automatically converted to camelCase. Any parameters which do
      * not match a setter on the target object are ignored.
      *
-     * @param mixed $target     The object to set parameters on
+     * @param ParameterizedInterface $target     The object to set parameters on
      * @param array $parameters An array of parameters to set
      */
-    public static function initialize($target, $parameters)
+    public static function initializeParameters(ParameterizedInterface $target, array $parameters = [])
     {
         if (is_array($parameters)) {
```

Comment:
Make sense if use an arrayable object is a real use case ? Typehint the parameters ensure the behavior.